shapely/constructive.py

```diff
@@ -508,12 +508,40 @@ def build_area(geometry, **kwargs):
 
 
 @multithreading_enabled
-def make_valid(geometry, **kwargs):
+def make_valid(geometry, method="linework", keep_collapsed=True, **kwargs):
     """Repairs invalid geometries.
 
+    Two ``methods`` are available:
+
+    * the 'linework' algorithm tries to preserve every edge and vertex in the input. It
+      combines all rings into a set of noded lines and then extracts valid polygons from
+      that linework. An alternating even-odd strategy is used to assign areas as
+      interior or exterior. A disadvantage is that for some relatively simple invalid
+      geometries this produces rather complex results.
+    * the 'structure' algorithm tries to reason from the structure of the input to find
+      the 'correct' repair: exterior rings bound area, interior holes exclude area.
+      It first makes all rings valid, then shells are merged and holes are subtracted 
+      from the shells to generate valid result. It assumes that holes and shells are
+      correctly categorized in the input geometry.
+
+    Example:
+
+    |make_valid_methods|
```

Comment:
And I _think_ you can also use `/_static/images/make_valid_methods.png` (i.e. absolute path, which means relative from the source root), then it doesn't depend on where this file lives (to ensure a correct relative path)

The `_static` subdirectory is not necessarily needed (that name is used by sphinx in the generated doc build, but we don't necessarily have to use that for the sources), it could also be just `/images`.

Reply:
I had a look at how the images are inserted in manual.rst and that's a really nice system... so I use that now.